scripts/upload.py

```diff
@@ -12,6 +13,35 @@ def get_unique_name(filename, unique_id) -> str:
         newname = "{0}-perf-lab-report.json".format(randint(1000, 9999))
     return newname
 
+def upload_and_queue(globpath, container, queue, sas_token_env, storage_account_uri):
+    try:
+        sas_token_env = sas_token_env
+        sas_token = os.getenv(sas_token_env)
+        if sas_token is None:
+            getLogger().error("Sas token environment variable {} was not defined.".format(sas_token_env))
+            return 1
+
+        files = glob(globpath, recursive=True)
+
+        for infile in files:
+            blob_name = get_unique_name(infile, os.getenv('HELIX_WORKITEM_ID'))
+
+            getLogger().info("uploading {} to storage account".format(infile))
+
+            blob_client = BlobClient(blob_url=storage_account_uri, container=container, blob=blob_name, credential=sas_token)
+            
+            with open(infile, "rb") as data:
```

Comment:
can we call `upload` below here instead of duplicating the code?

Reply:
Yeah, seems great. Not sure why I didn't do this in the first place.